UPGRADE.md

```diff
@@ -2,6 +2,45 @@
 
 ## dev-develop
 
+### Definition of security contexts
+
+The definition of security contexts in the `Admin` classes have changed. They
+used to look like the following example:
+
+```php
+public function getSecurityContexts()
+{
+    return [
+        'Sulu' => [
+            'Media' => [
+                'sulu.media.collections',
```

Comment:
I am not saying anything here about defaults :-) My idea is that that they should update that anyway ;-) It defaults to the value it had before, should I mention these?